omnigan/transforms.py

```diff
@@ -81,6 +83,83 @@ def __call__(self, data):
         return {task: torch.flip(tensor, [3]) for task, tensor in data.items()}
 
 
+class RandomRotations:
+    def __init__(self, p=0.5, angle=5):
+        self.p = p
+        self.angle = angle
+
+    def cut_balck_edge(self, rotated, angle_selected):
```

Comment:
so that means you crop so that we don't have the black corners, whatever the rotation is ?

Reply:
Indeed~